Bio/SeqIO/FastaIO.py

```diff
@@ -45,14 +49,44 @@ def SimpleFastaParser(handle):
     ('delta', 'CGCGC')
 
     """
-    # Skip any text before the first record (e.g. blank lines, comments)
-    for line in handle:
-        if line[0] == ">":
-            title = line[1:].rstrip()
-            break
-    else:
-        # no break encountered - probably an empty file
+    try:
+        line = next(handle)
+    except StopIteration:
         return
+    if not line.startswith(">"):
+        warnings.warn(
+            "Previously, the FASTA parser silently ignored comments at the\n"
+            "beginning of the FASTA file (before the first sequence).\n"
+            "\n"
+            "Nowadays, the FASTA file format is usually understood not to\n"
+            "have any such comments, and most software packages do not allow\n"
+            "them. Therefore, the use of comments at the beginning of a FASTA\n"
+            "file is now deprecated in Biopython.\n"
+            "\n"
+            "In a future Biopython release, this deprecation warning will be\n"
+            "replaced by a ValueError. To avoid this, there are three\n"
+            "options:\n"
+            "\n"
+            "(1) Modify your FASTA file to remove such comments at the\n"
+            "beginning of the file;\n"
+            "\n"
+            "(2) Use SeqIO.parse with the 'fasta-pearson' format instead of\n"
+            "'fasta'. This format is consistent with the FASTA format defined\n"
+            "by William Pearson's FASTA aligner software. Thie format allows\n"
+            "for comments before the first sequence; lines starting with the\n"
+            "';' character anywhere in the file are also regarded as comment\n"
+            "lines and are ignored.\n"
+            "\n"
+            "(3) Use the 'fasta-blast' format. This format regards any lines\n"
+            "starting with '!', '#', or ';' as comment lines. The\n"
+            "'fasta-blast' format may be safer than the 'fasta-pearson'\n"
+            "format, as it explicitly indicates which lines are comments.\n",
+            BiopythonDeprecationWarning,
```

Comment:
Can you leave out the explicit new lines except for the paragraph breaks? I think that would work better at the terminal.